EventListener/BodyListener.php

```diff
@@ -86,13 +86,13 @@ public function onKernelRequest(GetResponseEvent $event)
         $contentType = $request->headers->get('Content-Type');
         $normalizeRequest = $this->normalizeForms && $this->isFormRequest($request);
 
-        if ($this->isDecodeable($request)) {
-            $format = null === $contentType
-                ? $request->getRequestFormat()
-                : $request->getFormat($contentType);
-
-            $format = $format ?: $this->defaultFormat;
+        if (null === $contentType) {
+            $format = $request->getRequestFormat($this->defaultFormat);
```

Comment:
BTW it looks wrong to use ``getRequestFormat`` (that's not up to you of course).
I think this is dedicated to the response format not the request content.

@xabbuh what do you think about only keeping the ``getContentType`` check ?